symplyphysics/definitions/wave_equation_in_one_dimension.py

```diff
@@ -59,10 +59,12 @@
     Derivative(displacement(position, time), position, 2),
     Derivative(displacement(position, time), time, 2) / phase_speed**2,
 )
-"""
-:laws:symbol::
+r"""
+:code:`Derivative(u(x, t), (x, 2)) = (1/v^2) * Derivative(u(x, t), (t, 2))`
```

Comment:
A comment should be alaways added, when description is not auto-generated.
BTW I will add `any` dimension, to allow using our symbols instead of SymPy.